content/docs/styled-system/color-mode.mdx

```diff
@@ -80,36 +90,75 @@ export default theme
 > Remember to pass your custom `theme` to the `ChakraProvider`, otherwise your
 > color mode config won't be taken into consideration.
 
-#### Behavior of ColorMode
+#### Common Configurations
 
-The current hierarchy of how the color mode is defined is as follows:
+To help you define you desired behavior, here we define all common usage of the theme config :
 
-- if `useSystemColorMode` is true `system`-color will be used as default -
-  `initialColorMode` is the fallback if system color mode can't be resolved
+> **Tip** If you prefer, you can use this [playground](https://codesandbox.io/s/chakra-ui-color-mode-test-f5fcwr?file=/src/uiLayer.chakra/customTheme.ts) to try different values.
 
-- if `data-theme` prop is defined through e.g. `ColorModeScript` / after
-  modification/initial load of the colorMode this value will be used
+> **Note** The hook `useColorMode` let you update color mode of your app in every cases. [Learn more](#changing-color-mode)
+
+```tsx live=false
+// A.
+// System sets initial value.
+// App subscribes to system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'system',
+  useSystemColorMode: true,
+}
+
+// B.
+// System sets initial value.
+// App color mode is detached from system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'system',
+  useSystemColorMode: false,
+}
+
+// C.
+// You choose initial value.
+// App color mode is detached from system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'dark', // 'dark' | 'light'
+  useSystemColorMode: false,
+}
+
+
+// D.
+// You choose initial value.
+// App subscribes to system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'dark', // 'dark' | 'light'
+  useSystemColorMode: true,
+}
+```
+
+#### Behavior of ColorMode
+
+The current hierarchy of how chakra resolves the color mode:
 
-- if `colorModeManager` = `localStorage` and a value is defined for
-  `chakra-ui-color-mode` this will be used
+- Get the color mode value in the specified storage (localStorage or cookie manager or custom)
```

Comment:
Can we change this to: 
```suggestion
- Get the color mode value in the specified storage (localStorage, cookie manager or custom)
```

Reply:
Included in commit 1d1ec8bb6a4a658357504dbde6b8077c8ac6cfb0